spec/features/reviews/user_views_reviews_spec.rb

```diff
@@ -29,4 +29,30 @@
     expect(page).to have_content(review.description)
     expect(page).to have_content(review.user.email)
   end
+
+  scenario 'visitor sees 25 reviews per page' do
+   neighborhood = FactoryGirl.create(:neighborhood)
+   25.times do |n|
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.